### ISSUE TITLE
Seems that WriteCSV is losing some of its data

### ISSUE THREAD

**Issue by squillace91** [yukithm/json2csv#6]
I have noticed that some conversion (can't figure it out the pattern that is causing it) lose its data when converting the content to the csv file.

**Steps to reproduce:**

```
b := &bytes.Buffer{}
wr := json2csv.NewCSVWriter(b)
responses := []map[string]interface{}{} 

//populate responses

csvContent, err := json2csv.JSON2CSV(responses) // csvContent seems to be complete!
wr.WriteCSV(csvContent) 
wr.Flush()
b.Bytes()
```

At this point when I print the string of the bytes, I get 2 columns missing.

Seems that the issue happens here:
![image](https://user-images.githubusercontent.com/1086713/94770087-9a96fa80-0368-11eb-81e9-a0cb550a01b8.png)

Which doesn't make sense because the value is there
![image](https://user-images.githubusercontent.com/1086713/94770112-a97dad00-0368-11eb-91b1-a482da5a137f.png)
![image](https://user-images.githubusercontent.com/1086713/94770202-ddf16900-0368-11eb-908b-14cecd3ed2dd.png)

**Comment by yukithm** [comment]
Thank you for your report.

I have reproduced the problem.
It seems that the trailing whitespace of the key has been removed in the process somewhere.

```go
func TestKeyWithTrailingSpace(t *testing.T) {
	b := &bytes.Buffer{}
	wr := json2csv.NewCSVWriter(b)
	responses := []map[string]interface{}{
		{
			" A":  1,
			"B ":  "foo",
			"C  ": "FOO",
		},
		{
			" A":  2,
			"B ":  "bar",
			"C  ": "BAR",
		},
	}

	csvContent, err := json2csv.JSON2CSV(responses) // csvContent seems to be complete!
	if err != nil {
		t.Fatal(err)
	}
	wr.WriteCSV(csvContent)
	wr.Flush()

	got := b.String()
	want := `/ A,/B ,/C  
1,foo,FOO
2,bar,BAR
`

	if got != want {
		t.Errorf("Expected %v, but %v", want, got)
	}
}
```

```
--- FAIL: TestKeyWithTrailingSpace (0.00s)
    /Users/yukithm/projects/json2csv/csv_writer_test.go:40: Expected / A,/B ,/C  
        1,foo,FOO
        2,bar,BAR
        , but / A,/B,/C
        1,,
        2,,
```

B and C are trimmed.

```
expected keys: "/ A", "/B ", "/C  "
actual keys: "/ A", "/B", "/C"
```

**Comment by yukithm** [comment]
Fixed and release [v0.1.2](https://github.com/yukithm/json2csv/releases/tag/v0.1.2).
